src/commands/exec.yml

```diff
@@ -0,0 +1,18 @@
+description: Run a command with secret environment variables loaded from SecretHub
+parameters:
+  command:
+    type: string
+    description: Command to execute with secrets
+  version:
+    type: string
+    default: latest
+    description: Version of the SecretHub CLI
+  flags:
+    type: string
+    default: ''
+    description: Flags to pass to the `secrethub run` command
```

Comment:
Do we want to pass the flags separately, i.e. create a field for each flag?

Reply:
These fields are limited, no maps or lists are supported. And it's easier to add new fields later on when they do support these types than the other way around.